tools/site_cobble/rdl_pkg/listeners.py

```diff
@@ -78,6 +78,7 @@ class BaseListener(RDLListener):
     def __init__(self):
         self.prefix_stack = []
         self.known_types = []
+        self.reset_byte_known_type = {}
```

Comment:
Regarding the name here, there's nothing saying the register (and its reset value) would be a single byte. Maybe just `reset_known_types` instead? I also made it plural since this is dictionary that will hold a number of values.

Reply:
i think this is just a typo, it's a dictionary so I think I intended it to be reset_*by*_known_type since the known types are the dict keys. Will fix up.